resource/plugins/python/publisher/collectors/widget/nuke_movie_publisher_collector_options.py

```diff
@@ -23,27 +22,27 @@ class NukeMoviePublisherCollectorOptionsWidget(BaseOptionsWidget):
     auto_fetch_on_init = True
 
     @property
-    def media_path(self):
+    def movie_path(self):
         '''Return the media path from options'''
-        result = self.options.get('media_path')
+        result = self.options.get('movie_path')
         if result:
             result = result.strip()
             if len(result) == 0:
                 result = None
         return result
 
-    @media_path.setter
-    def media_path(self, media_path):
-        '''Store *media_path* in options and update widgets'''
-        if media_path is not None and len(media_path) > 0:
-            self.set_option_result(media_path, 'media_path')
+    @movie_path.setter
+    def movie_path(self, movie_path):
+        '''Store *movie_path* in options and update widgets'''
+        if movie_path and len(movie_path) > 0:
```

Comment:
```suggestion
        if movie_path:
```
Basically the `len(movie_path) > 0:` checks if movie_path == '' and the if movie_path already checks that.

Reply:
isn't the code commited?